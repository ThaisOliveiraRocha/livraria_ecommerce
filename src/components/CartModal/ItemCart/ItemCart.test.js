import React from "react";
import { shallow } from "enzyme";
import ItemCartConnected, {ItemCart} from "./";

const item = {
  imagem: "book1.png",
  titulo: "Ã‰ Assim Que Acaba",
  genero: "romance",
  autor: "Colleen Hoover",
  ano: "2018",
  situacao: "novo",
  preco: "40.00",
  qtd: 0
};

describe("<ItemCart />", () => {
  it("should render without crashing", () => {
    shallow(<ItemCartConnected item={item}/>);
  });
  it("should render basic infos", () => {
    const wrapper = shallow(<ItemCartConnected item={item}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
