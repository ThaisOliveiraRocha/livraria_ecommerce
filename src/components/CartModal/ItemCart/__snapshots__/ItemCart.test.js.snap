// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemCart /> should render basic infos 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
